spring-boot-admin-server/src/main/java/de/codecentric/boot/admin/server/services/IntervalCheck.java

```diff
@@ -52,6 +52,7 @@ public class IntervalCheck {
 
 	private final Function<InstanceId, Mono<Void>> checkFn;
 
+	@lombok.Getter
```

Comment:
Add getter to allow access from `StatusUpdateTrigger` to set timeout on interval check. This prevents stream overflows.